jetty-ee9/jetty-ee9-webapp/src/test/java/org/eclipse/jetty/ee9/webapp/StandardDescriptorProcessorTest.java

```diff
@@ -130,7 +131,18 @@ public void testDuplicateServletMappingsFromDescriptors(WorkDir workDir) throws
         wac.setBaseResourceAsPath(docroot);
         wac.setDescriptor(webXml.toUri().toURL().toString());
         wac.start();
-        assertEquals("other", wac.getServletHandler().getServletMapping("/").getServletName());
+        ServletMapping[] mappings = wac.getServletHandler().getServletMappings();
+        ServletMapping mapping = null;
+        for (ServletMapping m : mappings)
+        {
+            if (m.containsPathSpec("/"))
+            {
+                assertEquals(null, mapping);
+                mapping = m;
```

Comment:
Add `assertNull(mapping, "Found duplicate mapping")` before the `mapping = m;`

if `mapping` was already set, this is a test failure (you just found duplicate mappings).

Reply:
But this is _precisely_ what is there already, see line 140 which asserts that `mapping` must be null.